### PR TITLE
Make compatible with TextMate

### PR DESCRIPTION
**Pull request by aequitas**
This change will make this language usable as a TextMate Bundle.

I have used my name/email as contact for the Bundle. Feel free to change.

- Add UUID's to tmLanguage and tmPreferences to make them work with TextMate
- Add Build, Run, Fmt and Test tmCommands
- Run Fmt on save
- Symlink Comments.tmPreferences to TextMate location
- Add install instructions
- Add info.plist required by TextMate

### CODE REVIEW

File: Commands/Build.YAML-tmCommand [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+inputFormat: text
+keyEquivalent: '@b'
+name: Build
+outputCaret: afterOutput
+outputFormat: html
+outputLocation: newWindow
+requiredCommands:
+- command: zig
+  locations:
+  - /opt/local/bin/zig
```
Comment (daurnimator) [inline Commands/Build.YAML-tmCommand]: Where do these paths come from?

Comment (aequitas) [reply Commands/Build.YAML-tmCommand]: These are install locations for macos package managers like Brew and Macports. The paths are additional as the default PATH will also be searched afaik.

**Comment by aequitas** [comment]
I don't know of any standard patterns to ci this. I might try if I can ci this by comparing screenshots of rendered syntax highlighting. Or at least loading the app with the syntax file and check if there are no errors (like missing uuid).

What might be nice to have is a check to verify if all yaml files are converted to their counterparts. Do you know of any other tool to convert the files other than PackageDev? I tried to find what PackageDev uses underneath but was not able to find it yet.

**Comment by aequitas** [comment]
I added a Travis CI build which verifies if the bundle loads correctly in TextMate and if at least one of the commands can be executed to build a binary from Zig source. The result can be viewed here: https://travis-ci.com/aequitas/sublime-zig-language/builds/145422495

I first tried using Github actions but their environment does not support accessibility options required to interact with Textmate to trigger the build.

File: .github/workflows/textmate.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    steps:
+      - name: Install dependencies
+        run: |
+          brew cask install textmate
+          brew install coreutils zig
+      - name: Textmate load success
+        run: |
+          # "install" bundle
+          mkdir -p "$HOME/Library/Application Support/TextMate/Bundles/"
+          ln -s $PWD "$HOME/Library/Application Support/TextMate/Bundles/Zig.tmbundle"
```
Comment (daurnimator) [inline .github/workflows/textmate.yaml]: Should `$PWD` be in double quotes here?

Comment (aequitas) [reply .github/workflows/textmate.yaml]: It should indeed.

**Comment by emekoi** [comment]
@aequitas, sorry for not seeing this earlier, but thanks for the contribution! regarding your name and email i think that would be more of a question to ask andrew about.

**Pull request merged**